lib/generators/components/component_generator.rb

```diff
@@ -13,17 +13,24 @@ def create_component_file
 
     def create_erb_file
       return if options["skip_erb"]
-      create_file "app/components/#{name}/_#{name}.html.erb"
+      create_file "app/components/#{name}/_#{filename}.html.erb"
     end
 
     def create_css_file
       return if options["skip_css"]
-      create_file "app/components/#{name}/#{name}.css"
+      create_file "app/components/#{name}/#{filename}.css"
     end
 
     def create_js_file
       return if options["skip_js"]
-      create_file "app/components/#{name}/#{name}.js"
+      create_file "app/components/#{name}/#{filename}.js"
     end
+
+    private
+
+    def filename
+      name.split('/').last
+    end
+
```

Comment:
This extra empty line fails in rubocop. Sorry, I should set up CI at some point :)

Reply:
Sorry about that. I'm working on another PR and I'll try to appease Rubocop first this time :)